src/ExtensionClass/tests.py

```diff
@@ -39,9 +39,9 @@ def test_mixing():
     ...              O2.inheritedAttribute('x')(a[0]))
 
     >>> class C(Base):
-    ...   def __class_init__(self):
+    ...   def __class_init__(cls):
```

Comment:
It is similar to a classmethod, as it gets passed the class itself rather than an instance when the metaclass creates a new subclass. And before this changeset the metaclass constructor was turning it into a classmethod. But that didn't work the same as the C in the case I added a test for here, where __class_init__ is patched in to a base class after its construction (or in the case where it's inherited from a non-ExtensionClass base).